common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -77,37 +84,48 @@
             }
         ],
         "@typescript-eslint/consistent-type-definitions": [
-            "error"
+            "error",
+            "interface"
+        ],
+        "@typescript-eslint/consistent-type-imports": [
+            "warn",
+            {
+                "prefer": "type-imports"
+            }
         ],
         "@typescript-eslint/dot-notation": [
             "error"
         ],
-        "@typescript-eslint/explicit-function-return-type": [
+        "@typescript-eslint/eol-last": [
             "off"
         ],
+        "@typescript-eslint/explicit-function-return-type": [
+            "warn",
+            {
+                "allowConciseArrowFunctionExpressionsStartingWithVoid": false,
+                "allowDirectConstAssertionInArrowFunctions": true,
+                "allowExpressions": true,
+                "allowHigherOrderFunctions": true,
+                "allowTypedFunctionExpressions": true
+            }
+        ],
         "@typescript-eslint/explicit-member-accessibility": [
-            "off"
+            "error",
+            {
+                "accessibility": "explicit"
+            }
         ],
         "@typescript-eslint/explicit-module-boundary-types": [
-            "off"
-        ],
-        "@typescript-eslint/func-call-spacing": [
-            "off"
+            "error"
         ],
         "@typescript-eslint/indent": [
             "off"
         ],
-        "@typescript-eslint/interface-name-prefix": [
-            "off"
-        ],
-        "@typescript-eslint/keyword-spacing": [
+        "@typescript-eslint/linebreak-style": [
```

Comment:
I couldn't find documentation for `@typescript-eslint/linebreak-style`, just `linebreak-style`.  Does this rule actually exist?

Reply:
Not sure where this one is coming from. It may be a legacy rule that is included in the config I'm inheriting from. I don't override this rule myself.